blazar/plugins/devices/k3s_plugin.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+#
+# Author: Pierre Riteau <pierre@stackhpc.com>
```

Comment:
You can put Chameleon here if you want, doesn't matter that much since we won't contribute this back